plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -227,6 +227,9 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_MULTI_STATEMENT_WRITES:
                 return true;
 
+            case SUPPORTS_COMMENT_ON_VIEW:
```

Comment:
Can we extend `BaseConnectorTest#testCommentView` to verify that if SUPPORTS_COMMENT_ON_VIEW is supported then COMMENT ON TABLE doesn't work on views ?
I think COMMENT ON TABLE probably works on hive connector views at this point and it shouldn't when COMMENT ON VIEW is supported

Reply:
Engine should block it in my opinion. `CommentTask#commentOnTable` is lacking in such verification.

Reply:
Okay, could you create separate GH issue for that ?

Reply:
Sent #13161